opentelemetry-exporter-google-cloud/src/opentelemetry/exporter/cloud_trace/__init__.py

```diff
@@ -361,6 +361,32 @@ def _extract_resources(resource: Resource) -> Dict[str, str]:
     }
 
 
+LABELS_MAPPING = {
+    # "agent": "/agent",
+    "component": "/component",
+    "error.message": "/error/message",
+    "error.name": "/error/name",
+    "http.client_city": "/http/client_city",
+    "http.client_country": "/http/client_country",
+    "http.client_protocol": "/http/client_protocol",
+    "http.client_region": "/http/client_region",
+    "http.host": "/http/host",
+    "http.method": "/http/method",
+    "http.path": "/http/path",
+    "http.redirected_url": "/http/redirected_url",
+    # https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/http.md#common-attributes
+    "http.request_content_length": "/http/request/size",
+    "http.response_content_length": "/http/response/size",
```

Comment:
Please note that I used for those 2 a different mapping than the rest, could have been `http.request_size` but  I saw instead `http.request_content_length` which seems to fit nicely, not too sure about it though

Reply:
Ya it's not totally clear. I think this is fine for now until I can get clarification from the Cloud Trace folks